packages/common-ui-web/src/components/VehicleDynamicWireframe.tsx

```diff
@@ -0,0 +1,75 @@
+import { useMonkTheme } from '@monkvision/common';
+import { partSelectionWireframes } from '@monkvision/sights';
+import {
+  PartSelectionOrientation,
+  VehiclePart,
+  VehicleType,
+  VehicleTypeMapVehicleModel,
+} from '@monkvision/types';
+import { SVGProps } from 'react';
+import { DynamicSVG, DynamicSVGCustomizationFunctions } from './DynamicSVG';
+
+export interface VehicleDynamicWireframeProps {
+  vehicleType: VehicleType;
+  orientation?: PartSelectionOrientation;
+  onClickPart?: (parts: VehiclePart) => void;
+  getPartAttributes?: (part: VehiclePart) => SVGProps<SVGElement>;
+}
+
+const validElement = (element: SVGElement): boolean =>
+  element.id !== '' && element.classList.contains('car-part');
+
+function getOverlayAttributes(
+  onClickPart: NonNullable<VehicleDynamicWireframeProps['onClickPart']>,
+  getPartAttributes: NonNullable<VehicleDynamicWireframeProps['getPartAttributes']>,
+): NonNullable<DynamicSVGCustomizationFunctions['getAttributes']> {
+  return (element, groups) => {
+    const groupElement: (typeof groups)[number] | undefined = groups[0];
+    let part: VehiclePart;
+    if (groupElement && validElement(groupElement)) {
+      part = groupElement.id as VehiclePart;
+    } else if (validElement(element)) {
+      part = element.id as VehiclePart;
+    } else {
+      return { style: { display: 'none' } };
+    }
+    const attributes = getPartAttributes(part);
+    if (element.tagName === 'g') {
+      delete attributes.style;
+    }
```

Comment:
Why exactly do we need to remove the `style` property here? Can't we just leave it?

Reply:
no because for there are many styles can affect children. for example opacity .3 opacity both on g and path gives the path .5 opacity for the path. so only